### PR TITLE
Add ChemMaster 3000 at Virology back place

### PR DESCRIPTION
**Pull request by Varghh**
Title, add a ChemMaster 3000 at the back place of Viro so the virology don't need to pester the chemist to make his virus pills.

viroplace

:cl: Vargh
add: Adds a Chemmaster to Virology.
add: Adds an Office chair to Virology.
del: removed 2 stools from Virology.
/ :cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
I mean sure that's nice and all, but now the virologist can just stay inside his area indefinitely if they want. I kind of like the requirement for people to get help from others that those others to do their job and help

**Comment by Varghh** [comment]
One thing is getting help from others and another is getting inside someone else workplace to use his stuff, some chemist don't know how to make virus pills and can fuck up the mix with the new blood system

**Comment by SamHPurp** [comment]
Nice to have more people contributing, and well done on your first PR!

As for the contents of it, I don't think I can support it without more robust reasoning, it'll reduce interaction and opportunities for RP while at the same time buffing Viro and CMO antags (and anyone that wants to tear down a r-wall from maint).

**Comment by Citinited** [comment]
It's a chemmaster, not a chem dispenser, so is it that much of a buff?

In any case, yeah this would remove about the only reason virologists have to leave their lab. Overall probably a 👎

**Comment by Birdtalon** [comment]
Yeah it's a chem master not a dispenser so I don't really see any major balance concerns, only a small reducing of interaction.

**Comment by SamHPurp** [comment]
Oh welp, Master not dispenser. That's lessened my objections somewhat.

**Comment by Ar3nn** [comment]
Viro is a (mostly) thankless job with already little reason to interact, but honestly, Virologists have stigma from the crew because their work is so easily subverted and almost half the crew of any given shift will be blindly fearful of any virus you make, even if you and the CMO scream for the rest of the shift that there's no way it can harm them. I see no problem with them being shut up in their lab with the way the crew treats them and their creations anyway, there's almost no RP about it anyway, people powergame that it COULD be an assassination tool and therefore avoid it like the plague even when the CMO verifies the symptoms as beneficial. There's no interaction coming the other way, why force the one being ostracized to have to come out and be the one trying to interact when it's clear by the way the crew operates they don't want anything to do with them?

In short, I approve less reasons for the Viro to leave the lab, its not exactly a high-RP role in the first place. If you want RP go out of your lab and hold a conference in the holodeck or something with those opposed to your new plague and try to convince them it's good in person instead of over the radio.

**Comment by taukausanake** [comment]
Okay yeah, I thought this was a chem dispenser too. Chem master on the other hand seems fine and I guess for this case it is better for Viro to have their own rather than Chemistry. Interaction is good for departments and sub-departments but in this case something can go wrong so it's best to keep Viro contained as it should be.

Guess there's nothing wrong with being treated like a redheaded stepchild here 😜

**Comment by imsxz** [comment]
I approve of this personally, I've made viruses before with the chemist too lazy to properly name the pills (leaving them at "Blood (1u)") or flat out refusing to do this. It doesn't even remove any points of interaction either, the virologist will still need to ask someone for a chem bag to distribute the pills. It's not a massive buff, just makes it easier for the virologist to distribute an already completed virus or vaccine.

**Comment by Fox-McCloud** [comment]
After having discussed this, we decided against it.

We would like to encourage more interdepartmental co-operation, decisions, and gameplay, not more self-sufficiency.
